config/config.go

```diff
@@ -112,8 +112,9 @@ func Get() *ExportConfig {
 
 		options.AutomaticEnv()
 
-		if options.GetBool("Debug") {
-			options.Set("LogLevel", "DEBUG")
+		logLevel, present := os.LookupEnv("LOG_LEVEL")
+		if present {
+			options.Set("LogLevel", logLevel)
```

Comment:
I don't think this is necessary.  It looks like we already have a `LogLevel` env var that can be used to set the log level.  The default value is set to `INFO` at line 90.  So we should be able to set the log level by setting the `LOGLEVEL` env var without this change.

Nit picky:  I'm not a fan of jamming all the words together in the env var names (LOGLEVEL instead of LOG_LEVEL).  To me, breaking these up on word boundaries using an underscore makes them much easier to read at a glance.   It looks like we have a mixture of the approaches.  We probably need to pick an approach (used underscores or do not use underscores) and roll with it.  But that can be done in the future.  :)

Reply:
I went through and standardized the names, thanks for pointing that out! 

At first I didn't notice the `AutomaticEnv()` was pulling them based on the names. I left the `kubenv.GetString("Hostname")` alone, because I'm not sure where that's defined, and it looked like it was set as `Hostname` in other services. Everything else should match though.